adobe-esign-connector/webContent/layouts/frame-10-full-width.xhtml

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:f="http://xmlns.jcp.org/jsf/core"
+  xmlns:h="http://xmlns.jcp.org/jsf/html"
+  xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+  xmlns:ic="http://ivyteam.ch/jsf/component"
+  xmlns:p="http://primefaces.org/ui"
+  xmlns:pe="http://primefaces.org/ui/extensions">
+  
+<!--
+DESCRIPTION:
+This is the default template with right aligned labels to use the freya-ivy theme as well as iFrames approach.
+
+PORTAL:
+If you are using the Axon Ivy Portal, you can pass several parameters to it to use some features (e.g. process chain).
+You'll find the list of parameters here: https://developer.axonivy.com/portal/10.0/doc/portal-developer-guide/components/layout-templates.html#components-layout-templates-iframe-task-template
+For further information about iframes and usage in Portal please refer to: https://developer.axonivy.com/portal/10.0/doc/portal-developer-guide/iframe/index.html
+
+BRANDING:
+If you want to brand your dialogs, please take a look at our documentation: https://developer.axonivy.com/doc/10.0/designer-guide/user-interface/branding/index.html
+
+STYLING:
+If you want to add custom styles, you can simply add your own .css file (e.g. at the location: "webContent/layouts/styles/style.css")
+and refence it below in the head part.
+-->
```

Comment:
We think this file still be nice without these default hints :D (and 2 below also)